UPGRADE-5.4.md

```diff
@@ -66,6 +66,7 @@ SecurityBundle
 Security
 --------
 
+ * Deprecate `AuthenticationEvents::AUTHENTICATION_FAILURE`, use the `LoginFailureEvent` instead
```

Comment:
This event has been deprecated in 5.3, but we forgot to deprecate this constant.